### DISCUSSION TITLE
Cannot change the volume with the mouse wheel via the tray icon in Plasma (KDE)

### DISCUSSION THREAD

**Discussion by androos** [audacious-media-player/audacious#121] [General]
Operating System: Linux Mint 21.3
KDE Plasma Version: 5.24.7
KDE Frameworks Version: 5.98.0
Qt Version: 5.15.3
Kernel Version: 6.7.9-zabbly+ (64-bit)
Graphics Platform: X11
Processors: 4 × AMD A8-3500M APU with Radeon(tm) HD Graphics
Memory: 7.2 GiB of RAM
Graphics Processor: AMD SUMO
Kernel: 6.7.9-zabbly+ x86_64 bits: 64
Distro: Linux Mint 21.3 Virginia
Audacious 4.3.1

It works in mate without problems.

**Comment by jlindgren90** [comment]
This is an XEmbed vs. SNI (StatusNotifierItem) issue. I looked into it a few years ago, and I believe the issue lies on the Qt side (but my memory could be faulty).

**Reply by androos** [reply]
Does that mean you can't do anything about it? Where can I report this problem instead?

**Reply by radioactiveman** [reply]
Exactly, please report this to the Qt developers and reference this discussion. Thanks.

See also:
- https://redmine.audacious-media-player.org/issues/960 (your memory is not faulty, John :smiley:)
- https://github.com/qt/qtbase/blob/dev/src/gui/platform/unix/dbustray/qstatusnotifieritemadaptor.cpp#L145-L149 (still unsupported in Qt 6)

Edit: @birdie-github has reported this now as https://bugreports.qt.io/browse/QTBUG-123859
